progconcurrente/concurrente_teoria/index.html

```diff
@@ -1235,6 +1235,57 @@ <h2>
               el cual se encargará de revisar los procesos de compra y producción, eliminando así el código clonado,
               además de organizar los procesos desde el objeto de las bodegas en sí.</p>
 
+			<h2>Punto Muerto e Interbloqueo</h2>
+
+            <p>Se dice que un proceso está en punto muerto si se está esperando un evento que no ocurrirá. 
+				Interbloqueo por lo general implica varios procesos y puede conducir a la terminación del programa. 
+				El interbloqueo puede ocurrir cuando los procesos se comunican (por ejemplo, dos procesos intentar 
+				enviarse mensajes entre sí simultáneamente y sincrónicamente) pero es un problema más frecuentemente 
+				asociado con Administración de recursos. En este contexto son necesarios cuatro condiciones para que 
+				exista un interbloqueo:</p>
+
+                <ul>
+					<li>Los procesos deben reclamar el acceso exclusivo a los recursos.</li>
+  
+					<li>Los procesos deben contener algunos recursos mientras esperan otros (es decir,
+						adquirir recursos de forma fragmentaria).</li>
+  
+					<li>Los recursos no pueden eliminarse de los procesos en espera (sin preferencia).</li>
+  
+					<li>Existe una cadena circular de procesos en la que cada proceso tiene uno o más recursos requeridos por el siguiente proceso en la cadena.</li>
+				  </ul>
+
+				  <p>Las técnicas para evitar o recuperarse de un punto muerto se basan en negar al menos una de estas condiciones. 
+					Una de las mejores técnicas (aunque poco prácticas) para evitar interbloqueos es Algoritmo del banquero de Dijkstra. 
+					Dijkstra también planteó lo que se ha convertido en un clásico ejemplo ilustrativo en este campo, el de la Filósofos gastronómicos
```

Comment:
```suggestion
					Dijkstra también planteó lo que se ha convertido en un clásico ejemplo ilustrativo en este campo: el de los "filósofos gastronómicos".
```
Aquí se podría poner algún enlace externo, o bien mayor información, para saber a qué ejemplo se está haciendo referencia.